custom/icds_reports/static/js/directives/adhaar-beneficiary/adhaar-beneficiary.directive.js

```diff
@@ -2,10 +2,11 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function AdhaarController($scope, $routeParams, $location, $filter, demographicsService, locationsService,
-    userLocationId, storageService, haveAccessToAllLocations, baseControllersService) {
+    userLocationId, storageService, haveAccessToAllLocations, baseControllersService, isAlertActive) {
     baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
         userLocationId, storageService, haveAccessToAllLocations);
     var vm = this;
+    vm.isAlertActive = isAlertActive;
```

Comment:
Where is this used?

Reply:
It is used in the html templates to apply a style class for handling alert, example given below:

https://github.com/dimagi/commcare-hq/pull/25258/files#diff-9d5ea3110287b8e26b2d5cbac57e9333R2